### ISSUE TITLE
Subfolders not possible

### ISSUE THREAD

**Issue by duursma** [darold/external_file#5]
The documentation states that '/' and '\\' are forbidden for security reasons.  We are looking into transfering a huge archive of bfiles from oracle to postgres.  These bfiles are organised as a number of directories with in each directory a hierarchy 'env/year/month/file'.  I would expect that '..' would be forbidden instead of the path separator.

**Comment by darold** [comment]
Right, I have also encountered the problem and this is on my TODO list, it is also why I have not published a new release after the recent change.

**Comment by darold** [comment]
Commit b010347 fixes this issue.
